### PR TITLE
Repo access

### PR DESCRIPTION
**Pull request by lindseybateman**

### CODE REVIEW

File: content/docs/web-interface/adding-a-new-application.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+## Adding a new application
+
+For adding new applications on wercker we created a simple wizard.
+The wizard will take you trough a couple of guided steps to get the best setup
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: through

File: content/docs/web-interface/adding-a-new-application.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+from the get go.
+
+When you are logged in you can choose to create a new 'Application' from the
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: logged in, you can

COMMA na logged in

File: content/docs/web-interface/adding-a-new-application.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+create dropdown.
+
+![image](/images/adding-a-new-application_1.jpg)
+
+### Step 1: Choose a Git provider
+
+We first allow you to select your git provider, currently either GitHub or Bitbucket.
+If you’ve already authorized wercker to connect with either of these platforms,
+we will show this as well.
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: we will show you this as well.

YOU toevoegen

File: content/docs/web-interface/adding-a-new-application.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+![image](/images/adding-a-new-application_2.jpg)
+
+### Step 2: Select a repository
+
+We now present you with a list of repositories and also show if a repository is
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: repositories, and show

COMMA na repositories

File: content/docs/web-interface/adding-a-new-application.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+private, public, and if it’s a fork of another repo. We offer a input field
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: an input field

MET EEN N

File: content/docs/web-interface/adding-a-new-application.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+which you can use to filter your repositories, making selection even snappier.
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: making the selection even

THE toevoegen

File: content/docs/web-interface/adding-a-new-application.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+![image](/images/adding-a-new-application_3.jpg)
+
+In addition to private and public, we also show when a repository is already added
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: In addition to the private and public status of a repo, we also

File: content/docs/web-interface/adding-a-new-application.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+to wercker. When that is the case you can choose to join and become a collaborator
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: When that is the case,

COMMA toevoegen

File: content/docs/web-interface/adding-a-new-application.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+on that repository, when joining you will get `Build + view deploys` access rights
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: repository. When joining you

PUNT. NIEUWE ZIN

File: content/docs/web-interface/adding-a-new-application.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+to the app.
+
+When a repository is already added and is owned by an organization you are not
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: Omdraaien:

You are not allowed to join an application if the repository was already added by an organization.

File: content/docs/web-interface/adding-a-new-application.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+When you’re part of an owners team you can add applications to an organization.
+
+![image](/images/adding-a-new-application_4.jpg)
+
+### Step 4: Configure access
+
+Wercker uses an SSH key to checkout the code to run a build. For private repositories
+you need to add a public SSH key as a deploy key to your repository, or you can
+add this public SSH key to an user which has acces to the repository.
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: a user

GEEN N

Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: access

met een extra 's'

File: content/docs/web-interface/adding-a-new-application.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+If the `wercker.yml` is not present we will try to detect the programming language
+of your project and present a suggested wercker.yml file with sensible defaults
+that you need to commit and push to your repository.
+
+[Read more on the wercker.yml &rsaquo;](/docs/wercker-yml/index.html)
+
+![image](/images/adding-a-new-application_6.jpg)
+
+By default our `Docker stack` is enabled, there are differences in supported
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: is enabled. There are differences

PUNT NIEUWE ZIN

File: content/docs/web-interface/adding-a-new-application.md (hunk 1 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 2 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+environment to run your builds in.
+
+[Read more on the Docker and classic stack &rsaquo;](/docs/pipelines/stacks.html)
+
+### Step 6: Finishing up
+
+You are almost there! Set your application to `public` if you want to share it
+with the world. Public applications will only show your builds.
+
+When hittinh the `Finish` button you will be taken to the main application page.
```
Comment (mies) [inline content/docs/web-interface/adding-a-new-application.md]: Upon clicking the `Finish` button

File: content/docs/web-interface/adding-a-new-application.md (hunk 3 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

**Comment by mies** [comment]
Minor spelling and grammar. Looks good!

File: content/docs/web-interface/adding-a-new-application.md (hunk 4 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 5 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 6 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 7 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 8 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 9 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 10 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 11 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 12 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

File: content/docs/web-interface/adding-a-new-application.md (hunk 13 above)
Comment (lindseybateman) [inline content/docs/web-interface/adding-a-new-application.md]: check

**Pull request merged**